src/Microsoft.DotNet.Interactive.Http/AspNetExtensions.cs

```diff
@@ -82,12 +82,14 @@ public static IServiceCollection AddDotnetInteractiveHttpApi(this IServiceCollec
             {
                 r.Routes.Add(new VariableRouter(kernel));
                 r.Routes.Add(new KernelsRouter(kernel));
-                var htmlNotebookFrontedEnvironment = kernel.FrontendEnvironment as HtmlNotebookFrontedEnvironment;
```

Comment:
when renaming types because of spelling errors you should condier the feelings onf the person that did tath typo